### PR TITLE
Feature: FAQS & Contact Us screens

### LINKED ISSUE #80: Feature: Help Center Settings 10.101 - 10.103
### What does this pull request do?
Interfaces for the settings section screens

#### Description
- Help Center tabs
- FAQS
- Contact Us
- Sort FAQS

#### Tasks

- [x] Light theme UI for all the screens

### Screenshots

<div style="display: flex; flex-wrap: wrap;">

</div>

### PR DESCRIPTION
**Pull request by mr3nz1**
### What does this pull request do?
Interfaces for the settings section screens

#### Description
- Help Center tabs
- FAQS
- Contact Us
- Sort FAQS

#### Tasks

- [x] Light theme UI for all the screens

### Screenshots

<div style="display: flex; flex-wrap: wrap;">

</div>

### CODE REVIEW

**Review by Irirwanirira — APPROVED** [review]
Good Job

**Review by KeynesBizimana — APPROVED** [review]
Okay, Great!

**Review by Irirwanirira — APPROVED** [review]
Good Job

**Review by KeynesBizimana — APPROVED** [review]
Great !

File: components/UI/Search.tsx [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+            placeholder="Search"
+            placeholderTextColor={Colors.grayScale._400}
+            cursorColor={Colors.grayScale._500}
+            selectionColor={Colors.grayScale._100}
+            onChange={() => {
+              onchange();
+            }}
+          />
+          <SvgXml xml={FilterIcon} />
+        </View> */}
```
Comment (Nkbtemmy) [inline components/UI/Search.tsx]: Remove these commented codes; in case you are not using them

**Review by Nkbtemmy — CHANGES_REQUESTED** [review]

**Review by Nkbtemmy — APPROVED** [review]

**Pull request merged**